test/controllers/navigation/NavigatorTest.scala

```diff
@@ -39,7 +45,9 @@ class NavigatorTest extends WordSpec with Matchers with MockitoSugar with Export
 
   private val call = Call("GET", "url")
   private val config = mock[AppConfig]
-  private val navigator = new Navigator(config)
+  private val auditService = mock[AuditService]
+  private val hc: HeaderCarrier = HeaderCarrier(authorization = Some(Authorization(TestHelper.createRandomString(255))))
```

Comment:
Similar here

Reply:
Also given Audit Service is mocked you should be able to just use a mock HeaderCarrier if you want

Reply:
This has been addressed